worker/worker.go

```diff
@@ -840,7 +840,7 @@ func (w *worker) objectsHandlerGET(jc jape.Context) {
 		return
 	}
 
-	if strings.HasSuffix(path, "/") {
+	if strings.HasSuffix(jc.PathParam("path"), "/") {
```

Comment:
The only case where `path` and `jc.PathParam("path")` should be different in terms of the suffix is when the path is `"/"` right? 

So if this fix only related to this particular edge case why not do `if path == "" || strings.HasSuffix(path, "/")`?
This feels a bit more self explanatory to me.

Reply:
Ha! That was my first commit and I force-pushed over it.